w5e2.py

```diff
@@ -0,0 +1,37 @@
+#Cup of join
+''' Write a function called join, which receives an unlimited number of lists, each list as a parameter.
+ The function must be able to have another parameter called september.
+ The function must return one list consisting of all the lists received as parameters.
+ If the sep parameter is provided, it should be threaded as an element between any two lists. If not provided, the character "-" should be threaded in place.
+'''
+def join(*args, sep='-'):
+    if len(args) == 0:  # if no argument is provided
+        return None
+    list =[]
+    for arg in args:
+        for i in arg:
```

Comment:
try to use list comprehensions with two iterators for this

Reply:
I should add the separator between the lists How do I know in list comprehensions when does one loop end?

Reply:
new_list = [element for list in list_of_lists for element in list + [separator]]

Reply:
change the code inplace, dont comment the fixes